src/app/@ansyn/ansyn/modules/status-bar/components/display-panel/display-panel.component.less

```diff
@@ -32,6 +32,10 @@
 
 			i.icon {
 				margin: 0; // override default margin from container
+
+				&:before {
+					content: "\e902"; // override button icon: icon-screen-choice
```

Comment:
why not use `icon-screen-choice` class instead change the content

Reply:
@tltk90 
Use the class where?
The icon element is inside `ansyn-combo-box`. Not in my component.

Reply:
but you pass them to `ansyn-combo-box` through `ansyn-combo-box-option`

Reply:
@tltk90 

But I don't want to change the icons of the options. I only change the icon of the top button (.display-button)

Reply:
I think that the options should not be icons just text.

Reply:
@tltk90  I don't understand. For screens, the options are icons, no?

Reply:
I understand from Yuval that the icon only be in the button on the status-bar,
the option on the select menu will be a string

Reply:
What kind of string?

Reply:
good question.
I ask Yuval

Reply:
Ok. I don't understand it right.